archunit/src/main/java/com/tngtech/archunit/library/dependencies/SlicesRuleDefinition.java

```diff
@@ -67,12 +67,28 @@ public GivenSlices matching(String packageIdentifier) {
             return new GivenSlicesInternal(Priority.MEDIUM, Slices.matching(packageIdentifier));
         }
 
+        /**
+         * @see Slices#matching(String)
+         */
+        @PublicAPI(usage = ACCESS)
+        public GivenSlices matching(String packageIdentifier, Priority priority) {
+            return new GivenSlicesInternal(priority, Slices.matching(packageIdentifier));
+        }
+
         /**
          * @see Slices#assignedFrom(SliceAssignment)
          */
         @PublicAPI(usage = ACCESS)
         public GivenSlices assignedFrom(SliceAssignment assignment) {
             return new GivenSlicesInternal(Priority.MEDIUM, Slices.assignedFrom(assignment));
```

Comment:
This could likewise delegate to the other new method:
```suggestion
            return assignedFrom(assignment, Priority.MEDIUM);
```